langchain/document_loaders/browserless.py

```diff
@@ -0,0 +1,38 @@
+from typing import Iterator, List
+import requests
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+class BrowserlessLoader(BaseLoader):
+    """Loads the content of webpages using Browserless' /content endpoint"""
+    
+    def __init__(self, api_token: str, urls: List[str]):
+        """Initialize with API token and the URLs to scrape"""
+        self.api_token = api_token
+        """Browserless API token."""
+        self.urls = urls
```

Comment:
make `urls: Union[str, List[str]],` for [consistency](https://github.com/hwchase17/langchain/blob/8741e55e7c5ce8e389081f52dd949781269a6725/langchain/document_loaders/web_base.py#L61C11-L61C11).

Reply:
Done @rlancemartin

Reply:
thanks! testing now.